pkg/yamltemplate/metas.go

```diff
@@ -36,16 +36,12 @@ type CommentAndAnnotation struct {
 	Annotation *template.Annotation
 }
 
-type MetasOpts struct {
```

Comment:
for the record, the reason why i had MetasOps in this package is because yamltemplate may in theory have its own opts, and hence shouldnt have to expose underlying impl (template package's MetasOpts).

Reply:
Roger that.

We removed it (for now) because:
- to date, there has been no drift, and;
- `yamltemplate` is a concretion of `template` — the dependency is natural.

It was an abstraction that wasn't (yet) carrying its weight.